testdata/simple-go/models.go

```
@@ -0,0 +1,119 @@
+
+import (
+	"fmt"
+	"time"
+)
+
+// Constants for testing constant search
+const (
+	MaxUsers       = 100
+	DefaultTimeout = 30 * time.Second
+	ServiceVersion = "1.0.0"
+	DEBUG_MODE     = true
```

golangci-lint (1.64.8)

ST1003: should not use ALL_CAPS in Go names; use CamelCase instead

(stylecheck)